src/settings/__tests__/settingsReducer-test.js

```diff
@@ -24,6 +28,7 @@ describe('settingsReducer', () => {
         enable_offline_push_notifications: true,
         enable_online_push_notifications: true,
         enable_stream_push_notifications: true,
+        user_settings: undefined,
```

Comment:
(I think even after deleting the old properties from the example data, this test should still work fine with just this one line added -- because the test is already explicitly setting the properties it cares about, as it should.)